### DISCUSSION TITLE
Why does Homebrew install some formula with group "admin"?

### DISCUSSION THREAD

**Discussion by paulehoffman** [Homebrew/discussions#1954] [Everyday usage] [answered]
Greetings again. When I look through /usr/local/Cellar, I most of the directories are owned by me, with the group being my normal group. Some, however, are owned by me but have the group set to "admin" (group 80). Why is that? In the current case, I am absolutely sure I am logged in as me.

**Comment by SMillerDev** [comment, marked as answer]
If you run `sudo brew services start` it'll change ownership of all affected files to root/admin to prevent modification.

**Reply by paulehoffman** [reply]
Interesting, thanks! Does the fact that it does this only for `m4` on my system (which has a good handful of formula that I've added) indicate that `m4` would be more dangerous to have modified than other formulae?
